backend/run/print_queues.py

```diff
@@ -14,6 +14,7 @@
     'host': opts['redis_host'],
     'port': opts['redis_port'],
     'db': opts['redis_db'],
+    'password': opts['redis_pwd'],
```

Comment:
redis_password?

Reply:
Sorry for the mess, the option key in configfile is `redis_password`, but in `opts` object attr name is `redis_pwd`, see [code](https://github.com/fedora-copr/copr/pull/2762/files#diff-20fa3ce7e615864b4b1ed869c8f7547b60c693f52cf5be7cf6674b7b460e6ab6R291-R293)

Reply:
Make sense :)